### PR TITLE
Edit Home Page

### PR DESCRIPTION
**Pull request by anhthuOE27**
editlan2

### CODE REVIEW

**Comment by anhthuOE27** [comment]
ready

File: config/settings.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+en: "en"
+vi: "vi"
```
Comment (vanvtt-0952) [inline config/settings.yml]: thêm end-line cho file này

**Comment by anhthuOE27** [comment]
ready

**Comment by chienpv-0644** [comment]
conflicted

**Comment by anhthuOE27** [comment]
ready

**Pull request merged**